source/Libraries/GogLibrary/GogGameController.cs

```diff
@@ -55,10 +55,31 @@ public override void Dispose()
 
         public override void Install(InstallActionArgs args)
         {
+            if (HandleExtras())
+            {
+                return;
+            }
+
             InitiateInstall();
             StartInstallWatcher();
         }
 
+        private bool HandleExtras()
+        {
+            var shellLink = Game.Links.FirstOrDefault(x => x.Name == "shell:open");
+            if (shellLink == null)
+            {
+                return false;
+            }
+
+            Dispose();

```

Comment:
Do not call. Dispose is called by Playnite when Playnite is done with the controller.